templates/dovecot.conf.j2

```diff
@@ -0,0 +1,239 @@
+	unix_listener auth-userdb {
+		mode = 0660
+		group = {{ dovecot_vmail_group }}
+	}
+}
+
+# Maildir
+mailbox_list_index = yes
+mail_prefetch_count = 10
+
+{% if dovecot_dsync %}
+# dsync
+mail_plugins = $mail_plugins notify replication
+replication_max_conns = {{ dovecot_dsync_max_connections }}
+doveadm_port = {{ dovecot_dsync_port }}
+doveadm_password = {{ dovecot_dsync_password }}
+replication_dsync_parameters = {{ dovecot_dsync_parameters }}
+{% if dovecot_dsync_sync_timeout != None %}
+plugin {
+	replication_sync_timeout = {{ dovecot_dsync_sync_timeout }}
+}
+{% endif %}
+{% if dovecot_dsync_target != None %}
+plugin {
+	mail_replica = tcps:{{ dovecot_dsync_target }}
+}
+
+service replicator {
+	process_min_avail = 1
+	user = {{ dovecot_vmail_user }}
+	unix_listener replicator-doveadm {
+		mode = 0666
+	}
+}
+{% endif %}
+service doveadm {
+	inet_listener {
+		port = {{ dovecot_dsync_port }}
+		ssl = yes
+	}
+	unix_listener doveadm-server {
+		mode = 0600
+		user = {{ dovecot_vmail_group }}
+	}
+}
+service aggregator {
+	fifo_listener replication-notify-fifo {
+		user = {{ dovecot_vmail_user }}
+	}
+	unix_listener replication-notify {
+		user = {{ dovecot_vmail_user }}
+	}
+}
+{% endif %}
+
+# ACLs
+mail_plugins = $mail_plugins acl
+plugin {
+	acl = vfile:/etc/dovecot/dovecot-acl
+	acl_globals_only = yes
+	acl_user = %u
+}
+
+# IMAP
+login_greeting = IMAP ready.
+imap_client_workarounds = tb-extra-mailbox-sep
+imap_hibernate_timeout = {{ dovecot_imap_hibernate_timeout }}
+service imap-login {
+	process_min_avail = {{ dovecot_imap_min_procs }}
+}
+service imap {
+	unix_listener imap-master {
+		user = $default_internal_user
+	}
+}
+service imap-hibernate {
+	unix_listener imap-hibernate {
+		mode = 0660
+		group = $default_internal_group
+	}
+}
+protocol imap {
+	mail_plugins = $mail_plugins imap_zlib imap_sieve imap_acl
+
+	ssl_cert = <{{ dovecot_cert }}
+	ssl_key = <{{ dovecot_key }}
+
+	# Do autoexpunge only for IMAP
+	namespace private {
+{% for name,conf in dovecot_private_mailboxes.items() if 'autoexpunge' in conf %}
+		mailbox {{ name }} {
+			autoexpunge = {{ conf.autoexpunge }}
+		}
+{% endfor %}
+	}
+}
+
+# LDA
+protocol lda {
+	mail_plugins = $mail_plugins sieve
+	log_path = /dev/stderr
+	info_log_path = /dev/stderr
+}
+
+# sieve
+protocol sieve {
+	ssl_cert = <{{ dovecot_cert }}
+	ssl_key = <{{ dovecot_key }}
+
+	managesieve_implementation_string = sieved
+}
+plugin {
+	sieve = file:{{ dovecot_mail_location }}/private/%u/sieve;active={{ dovecot_mail_location }}/private/%u/dovecot.sieve
+	sieve_user_log = {{ dovecot_mail_location }}/private/%u/sieve.log
+	sieve_pipe_bin_dir = {{ dovecot_mail_location }}/sieve-pipe
+	sieve_before = {{ dovecot_mail_location }}/sieve/before.sieve
+
+	# Extensions and plugins
+	sieve_plugins = sieve_imapsieve sieve_extprograms
+	sieve_extensions = +spamtest +spamtestplus
+	sieve_global_extensions = +vnd.dovecot.pipe
+
+	# Spamtest
+	sieve_spamtest_status_type = score
+	sieve_spamtest_max_value = 15.0
```

Comment:
Should this be configurable? Or is a generally defined max value?

Reply:
It's the maximum for rspamd